rs/rosetta-api/icp/client/src/lib.rs

```diff
@@ -710,6 +760,60 @@ impl RosettaClient {
         )
         .await
     }
+
+    /// If a neuron is in the state NOT_DISSOLVING you start the dissolving process witht his function.
```

Comment:
What is the expected behavior when we call this and the state is already `DISSOLVING`?

Reply:
Added test for that